File: xxf_arch/src/main/java/com/xxf/arch/core/Logger.java
package com.xxf.arch.core;

/**
 * @Author: XGod  xuanyouwu@163.com  17611639080  https://github.com/NBXXF     https://blog.csdn.net/axuanqq  xuanyouwu@163.com  17611639080  https://github.com/NBXXF     https://blog.csdn.net/axuanqq
 * @Description
 */
public interface Logger {

    boolean isLoggable();

    void d(String msg);

    void d(String msg, Throwable tr);

    void e(String msg);

    void e(String msg, Throwable tr);
}

File: xxf/src/main/java/com/xxf/arch/core/Logger.kt
package com.xxf.arch.core

/**
 * @Author: XGod  xuanyouwu@163.com  17611639080  https://github.com/NBXXF     https://blog.csdn.net/axuanqq  xuanyouwu@163.com  17611639080  https://github.com/NBXXF     https://blog.csdn.net/axuanqq
 * @Description
 */
interface Logger {
    val isLoggable: Boolean
    fun d(msg: String?)
    fun d(msg: String?, tr: Throwable?)
    fun e(msg: String?)
    fun e(msg: String?, tr: Throwable?)
}